pipelines/deployer/deployer.yaml

```diff
@@ -120,7 +121,8 @@ generate_namespace_values: &generate_namespace_values
       cd config/${PATH_TO_NAMESPACES}
       echo "creating helm compatible values file from namespace data"
       echo 'namespaces:' > $namespace_values_file
-      for ns in *; do
+      for ns in gsp-* "${CLUSTER_NAME}-*"; do
```

Comment:
Don't think globs work like this here 😭 ... did you try it? ... I just get the list of globs back

Might have to do something horrible like:

`for ns in $(ls . | grep -E "^(gsp|${CLUSTER_NAME})-"); do`

Reply:
```
for glob in gps-* sandbox-*
do
    for ns in "${glob}"
    do
        ls -l "${ns}"
    done
done
```
:trollface:

Reply:
touché